stefanpenner/country_select
# Triage Travis CI configuration

- Use latest rubies for travis
- Switch to travis-handled rbx version; add dependent gems
- Use latest appraisal gem that supports Bundler platforms
- Update appraisal gemfiles to use latest appraisal
- Add fragile rubies to allowable failures in travis-ci

Rubinius support in Travis CI was really hard to nail down, and now it looks like `appraisal` is doing some weird things on Travis with `rake`. I'm thinking we should allow rbx failures for now. What do you think @stefanpenner?
